registry/storage/driver/base/base.go

```diff
@@ -155,7 +164,9 @@ func (base *Base) List(ctx context.Context, path string) ([]string, error) {
 		return nil, storagedriver.InvalidPathError{Path: path, DriverName: base.StorageDriver.Name()}
 	}
 
+	start := time.Now()
 	str, e := base.StorageDriver.List(ctx, path)
+	prometheus.BlobAction.WithValues("list").UpdateSince(start)
```

Comment:
Can these somehow include the name of the driver that is in use? That would be helpful for comparing driver implementations in production.